homeassistant/components/homewizard/sensor.py

```diff
@@ -69,35 +69,41 @@ def to_percentage(value: float | None) -> float | None:
         key="smr_version",
         translation_key="dsmr_version",
         entity_category=EntityCategory.DIAGNOSTIC,
-        has_fn=lambda data: data.protocol_version is not None,
-        value_fn=lambda data: data.protocol_version,
+        has_fn=lambda data: data.measurement.protocol_version is not None,
+        value_fn=lambda data: data.measurement.protocol_version,
     ),
     HomeWizardSensorEntityDescription(
         key="meter_model",
         translation_key="meter_model",
         entity_category=EntityCategory.DIAGNOSTIC,
-        has_fn=lambda data: data.meter_model is not None,
-        value_fn=lambda data: data.meter_model,
+        has_fn=lambda data: data.measurement.meter_model is not None,
+        value_fn=lambda data: data.measurement.meter_model,
     ),
     HomeWizardSensorEntityDescription(
         key="unique_meter_id",
         translation_key="unique_meter_id",
         entity_category=EntityCategory.DIAGNOSTIC,
-        has_fn=lambda data: data.unique_id is not None,
-        value_fn=lambda data: data.unique_id,
+        has_fn=lambda data: data.measurement.unique_id is not None,
+        value_fn=lambda data: data.measurement.unique_id,
     ),
     HomeWizardSensorEntityDescription(
         key="wifi_ssid",
         translation_key="wifi_ssid",
         entity_category=EntityCategory.DIAGNOSTIC,
-        has_fn=lambda data: data.wifi_ssid is not None,
-        value_fn=lambda data: data.wifi_ssid,
+        has_fn=(
+            lambda data: data.system is not None and data.system.wifi_ssid is not None
+        ),
+        value_fn=(
+            lambda data: data.system.wifi_ssid if data.system is not None else None
+        ),
     ),
     HomeWizardSensorEntityDescription(
         key="active_tariff",
         translation_key="active_tariff",
-        has_fn=lambda data: data.tariff is not None,
-        value_fn=lambda data: None if data.tariff is None else str(data.tariff),
+        has_fn=lambda data: data.measurement.tariff is not None,
+        value_fn=lambda data: None
+        if data.measurement.tariff is None
+        else str(data.measurement.tariff),
```

Comment:
```suggestion
        has_fn=lambda data: data.measurement.tariff is not None,
        value_fn=(
            lambda data: None
            if data.measurement.tariff is None
            else str(data.measurement.tariff)
        ),
```
put brackets around multi line statements please

Reply:
Ah, that multi line stuff was done by Ruff in the first place, IMO adding brackets should also be enforced by this.

Thank you for your suggestion!